src/wireviz/Harness.py

```diff
@@ -336,66 +326,54 @@ def bom(self):
             designators.sort()
             total_length = sum(i.length for i in items.values())
             cable_type = f', {shared.type}' if shared.type else ''
-            gauge_name = f' x {shared.gauge} {shared.gauge_unit}'if shared.gauge else ' wires'
+            gauge_name = f' x {shared.gauge} {shared.gauge_unit}' if shared.gauge else ' wires'
             shield_name = ' shielded' if shared.shield else ''
             name = f'Cable{cable_type}, {shared.wirecount}{gauge_name}{shield_name}'
-            item = {'item': name, 'qty': round(total_length, 3), 'unit': 'm', 'designators': designators}
-            if shared.manufacturer is not None:  # set manufacturer only if it exists
-                item['manufacturer'] = shared.manufacturer
-            if shared.manufacturer_part_number is not None:  # set part number only if it exists
-                item['manufacturer part number'] = shared.manufacturer_part_number
-            if shared.internal_part_number is not None:  # set part number only if it exists
-                item['internal part number'] = shared.internal_part_number
+            item = {'item': name, 'qty': round(total_length, 3), 'unit': 'm', 'designators': designators,
+                    'manufacturer': shared.manufacturer, 'manufacturer part number': shared.manufacturer_part_number, 'internal part number': shared.internal_part_number}
             bom_cables.append(item)
         # bundles (ignores wirecount)
         wirelist = []
         # list all cables again, since bundles are represented as wires internally, with the category='bundle' set
-        bundle_group = lambda b: (b.category, b.type, b.gauge, b.gauge_unit, b.length) # TODO: Why is b.length included?
```

Comment:
@kvid I think this was your question... right now I guess it's not important to have `length` included in the check, but see #56 , potentially this might become relevant in the future.

Reply:
As long as the wires can be grouped across bundles, I don't see the need to group bundles at all. If needing the option to have cut wires in the BOM instead of total length, then that will affect only the wire grouping, as far as I can see. I'm planning a PR with such a simplification.